Projects/Forever/GraphQL/Octopus/MemberReferralInformationCodeUpdate.graphql

```diff
@@ -0,0 +1,26 @@
+mutation MemberReferralInformationCodeUpdate($code: String!) {
+  memberReferralInformationCodeUpdate(code: $code) {
+    referralInformation {
```

Comment:
we don't use this referralInformation in code. so you can remove it.
We only use userError